apps/challenges/views.py

```diff
@@ -3354,3 +3354,103 @@ def pwc_task_dataset(request):
             "error": "You are not authorized to make this request. Please ask EvalAI admin to add you as a staff user for accessing this API."
         }
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
+
+
+@api_view(["GET", "PATCH", "DELETE"])
+@throttle_classes([UserRateThrottle])
+@permission_classes(
+    (
+        permissions.IsAuthenticatedOrReadOnly,
```

Comment:
Why `ReadOnly`?

Reply:
Yeah, it should be `IsAuthenticated` only.